formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/internal/Helpers.kt

```diff
@@ -22,8 +22,15 @@ internal enum class ProtoWireType(val typeId: Int) {
     ;
 
     companion object {
+        private val entryArray = Array(8) { typeId ->
```

Comment:
My bad, the size is chosen based on the fact the only call site explicitly zero all the bits except three least significant.

Perhaps, as there's only one call site, explicitly move this operation inside `from` and rename `from` to `fromLowerBits`. WDYT?